### DISCUSSION TITLE
Routed long messages appear twice.

### DISCUSSION THREAD

**Discussion by gulfaraz** [folke/noice.nvim#939] [Q&A]
[My kickstart.nvim config.](https://github.com/gulfaraz/kickstart.nvim/blob/f74d5f74ba2ff894070e1aa5e0d611b8f8a0fbd3/lua/custom/plugins/init.lua#L11-L17)

I added [a route](https://github.com/gulfaraz/kickstart.nvim/blob/f74d5f74ba2ff894070e1aa5e0d611b8f8a0fbd3/lua/custom/plugins/init.lua#L11-L17) to the `noice` config,

```
{
    view = 'vsplit',
    filter = { event = 'msg_show', min_height = 20 },
    opts = { stop = true },
},
```

It routes long messages to a new vertical buffer. However, I still see the default notification which disappears after some time.

I would like all long messages to be in a `vsplit` so I can read the whole message.

I tried various combinations of `skip = true` and `redirect` attempts to remove the default notification for long messages but it always appears. `stop = true` does not seem to affect the default notification.

**How to make the long messages only appear in the vertical buffer?**

I also noticed that `TSInstallInfo` outputs in to the default notification despite being longer than 20 lines. **Is it possible to show this in a vertical buffer instead?**

Thank You!

https://github.com/user-attachments/assets/52555bb6-389d-46e2-8477-c563f2c9380a

**Comment by YiHui-Liu** [comment]
same issue for me, the route does not work with `TSInstallInfo`
